src/main.ts

```diff
@@ -9,6 +9,9 @@ async function bootstrap() {
     app.get<IConfigurationService>(IConfigurationService);
   const applicationPort: string =
     configurationService.getOrThrow('applicationPort');
+
+  app.enableCors();
```

Comment:
This is unrelated but it's needed when testing the service in combination with a locally running frontend.

Reply:
I'd suggest we make this change in a separate PR.